lib/schemas/mental_status.js

```diff
@@ -0,0 +1,114 @@
+module.exports = {
+  "id": "mental_status",
+  "type": "array",
+  "properties": {
+    "text": {
+      "type": "string"
+    },
+    "identifiers": {
+      "type": "string",
+      "items": {
+        "$ref": "cda_id"
+      }
+    },
+
+    "observation": {
+      "type": "array",
+      "properties": {
```

Comment:
Need an "items" specifier here

Reply:
I can see what the significance is, b ut still no change in output